matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/PatientMatch.java

```diff
@@ -341,7 +341,6 @@
      *                  "emails":["patcat@catrescuecentre.kom"]
      *                },
      *         "cc":  ["mary@blueberry.com"],
-     *         "subjectPatientId":"P000005"
```

Comment:
I thought we decided to keep this information for potential future use? If lost, in some cases it can't be recovered

Reply:
We don't use it now, I'm not sure how useful is it to keep.